### PR TITLE
HW for 7/30/22

### PR DESCRIPTION
**Pull request by awesomeprogrammer1**
# what

HW to be Completed by 7/30/22

### CODE REVIEW

File: code_school/functions.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    area = width * height; perimeter = (width+height)*2 ; result = f'P = {perimeter}, Area = {area}' ; return result
+print(print_perimeter_area_rectangle(5,4))
+
+
+def perimeter_area_rectangle_dict(width, height):
+    area = width * height 
+    perimeter = (width+height)*2  
+    dict[area] = perimeter
+    result = dict
+    return result
```
Comment (sryabkov) [inline code_school/functions.py:46]: Not quite... Try again.

Hint: look at `functions_test.py`

File: code_school/functions.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+
+def print_perimeter_area_rectangle_dict(dict):
```
Comment (sryabkov) [inline code_school/functions.py:49]: You need to implement this function

**Review by sryabkov — CHANGES_REQUESTED** [review]
Look at inline comments

Before you submit, you need to run `python3 -m pytest` in the `code_school` directory and make sure you get no errors

File: code_school/functions.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+# def strlength(a):
+#     return len(a)
+
+
+# print(strlength(input('Write Anything: ')))
+
+# P= 123, Area=123
+dict = {}
+def print_perimeter_area_rectangle(width, height):
+    area = width * height; perimeter = (width+height)*2 ; result = f'P = {perimeter}, Area = {area}' ; return result
```
[the remark this answers is not in the document]
Comment (awesomeprogrammer1) [reply code_school/functions.py:37]: Fixed

File: code_school/functions.py (hunk 1 above)
Comment (awesomeprogrammer1) [reply code_school/functions.py:46]: Fixed assignment

File: code_school/functions.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+# print(strlength(input('Write Anything: ')))
+
+# P= 123, Area=123
+dict = {}
+
+
+def print_perimeter_area_rectangle(width, height):
+    result = f"P = {(width+height)*2}, Area = {width * height}"
+    return result
```
Comment (sryabkov) [inline code_school/functions.py:40]: Should be one line, not two...

File: code_school/functions.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+
+print(print_perimeter_area_rectangle(5, 4))
+
+
+def perimeter_area_rectangle_dict(width, height):
+    dict["area"] = width * height
+    dict["perimeter"] = (width + height) * 2
+    result = dict
+    return result
```
Comment (sryabkov) [inline code_school/functions.py:50]: Do the tests pass?

**Review by sryabkov — CHANGES_REQUESTED** [review]
See inline comments... 

and RUN THE TESTS before submitting!

File: code_school/functions.py (hunk 5 above)
Comment (awesomeprogrammer1) [reply code_school/functions.py:50]: Test number one has passed, test number two has failed

Comment (sryabkov) [reply code_school/functions.py:50]: I don't believe you: both tests fail on my machine

File: code_school/functions.py (hunk 4 above)
Comment (awesomeprogrammer1) [reply code_school/functions.py]: Fixed

File: code_school/functions.py (hunk 5 above)
Comment (awesomeprogrammer1) [reply code_school/functions.py]: Both tests passed

File: code_school/functions.py (hunk 2 above)
Comment (awesomeprogrammer1) [reply code_school/functions.py]: Implemented

File: code_school/functions.py [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+def print_perimeter_area_rectangle(width, height):
+    return f"Perimeter = {(width+height)*2}, Area = {width * height}"
+    ""
+
+
+print(print_perimeter_area_rectangle(5, 4))
+
+
+def perimeter_area_rectangle_dict(width, height):
+    dict["Area"] = width * height
```
Comment (sryabkov) [inline code_school/functions.py:47]:
```suggestion
    dict = {}
    dict["Area"] = width * height
```

**Review by sryabkov — COMMENTED** [review]
Looks OK, see a couple of minor comments

File: code_school/functions.py [hunk 7]
```diff
@@ -36,16 +36,17 @@
 
 
 def print_perimeter_area_rectangle(width, height):
-    result = f"P = {(width+height)*2}, Area = {width * height}"
+    result = f"Perimeter = {(width+height)*2}, Area = {width * height}"
     return result
+    ""
```
Comment (sryabkov) [inline code_school/functions.py:40]:
```suggestion
```

What is this?

**Review by sryabkov — APPROVED** [review]

File: code_school/functions.py (hunk 7 above)
Comment (awesomeprogrammer1) [reply code_school/functions.py:40]: Fixed

File: code_school/functions.py (hunk 6 above)
Comment (awesomeprogrammer1) [reply code_school/functions.py:48]: Fixed

**Pull request merged**